common/djangoapps/student/models.py

```diff
@@ -877,6 +879,46 @@ def num_enrolled_in(self, course_id):
 
         return enrollment_number
 
+    def num_enrolled_in_exclude_admins(self, course_id):
+        """
+        Returns the count of active enrollments in a course excluding admins and staff.
+
+        Arguments:
+            course_id (CourseLocator): course_id to return enrollments (count)
+
+        Returns:
+            number: Count of enrollments excluding staff and admin
+
+        """
+        # To avoid circular imports.
+        from roles import CourseInstructorRole, CourseStaffRole
+        course_locator = course_id
+
+        if getattr(course_id, 'ccx', None):
+            course_locator = course_id.to_course_locator()
+
+        staff = CourseStaffRole(course_locator).users_with_role()
+        admins = CourseInstructorRole(course_locator).users_with_role()
```

Comment:
Instead of filtering the CCX coach in the code below why not use the `CourseCcxCoachRole` to filter the results like you do for `staff` and `admins`?

cc @giocalitri who might be able to tell me if this is a reasonable way to do this